nss-cohort-36/bangazon-workforce-mgt-lords-of-might-magic
# Jc department details

# Description

Department details now displays all employees in the department.

Closes #[Enter issue #]

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jc-department-details`
4. go into departments, then choose a department it should show the department and the employees in it.

# Checklist:
